recipe/run_conda_forge_build_setup_osx

```diff
@@ -12,6 +12,13 @@ export PYTHONUNBUFFERED=1
 #   way is working well enough.
 export MACOSX_DEPLOYMENT_TARGET="10.9"
 
+if [ "$CIRCLECI" == "true" ]; then
+    echo "Downloading sdk"
+    curl -L -O https://github.com/phracker/MacOSX-SDKs/releases/download/10.13/MacOSX10.9.sdk.tar.xz
```

Comment:
Another option would be to package our own copy of the SDK. This way we could just always install that conda package and use it regardless of what CI we are on.

xref: https://github.com/conda-forge/staged-recipes/pull/5382

Reply:
Since we don't know whether we can distribute it as a conda package, I'd go with this option.